guides/v2.1/security/two-factor-authentication.md

```diff
@@ -0,0 +1,106 @@
+
+At this time, Two-Factor Authentication can be installed only from the command line.
+
+Two-Factor Authentication gives you the ability to:
+
+- Enable authenticator support for the Admin.
+- Manage and configure authenticator settings globally or per user account.
+- Reset authenticators and manage trusted devices for users.
+
+At this time, Two-Factor Authentication can be installed only from the command line.
+
+{:.bs-callout .bs-callout-info}
+**Magento Community Contribution** - Magento thanks [Riccardo Tempesta](https://twitter.com/rictempesta) of [MageSpecialist](https://partners.magento.com/portal/details/partner/id/129) for contributing these features as part of the Magento Community Engineering program.
+
+## Install 2FA
+
+Install the extension using the following Composer command:
+
+``` bash
+composer require msp/twofactorauth:3.0.0
+```
+
+To complete installation in an existing Magento instance, run the following commands to enable the module:
+
+``` bash
+php bin/magento module:enable --all
+php bin/magento setup:upgrade
+```
+
+## Configure and manage 2FA
+
+See the Magento Admin User Guide to [configure](https://docs.magento.com/m2/ee/user_guide/stores/security-two-factor-authentication.html) 2FA settings and [manage user authenticators](https://docs.magento.com/m2/ee/user_guide/stores/security-two-factor-authentication-manage.html).
+
+Administrators have options to:
+
+* Review existing authenticators configured per user account
+* Require specific authenticators
+* Reset or remove authenticators to resolve access issues
+* Revoke access for devices to resolve access issues
+
+## Install authenticator
+
+After enabling and configuring 2FA, your staff need to:
+
+1.	Install one of the supported authenticators to a mobile device or obtain a configured device.
+
+    For example, you can install the Google Authenticator app to a mobile device such as a smart phone or tablet. Depending on the OS, you can download and install the authenticator from [Google Play](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en) or [iOS App Store](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DiOS&hl=en&oco=0).
+
+1.	Log into the Magento Admin and configure their authenticator. The first login prompts to configure the authenticator. For complete instructions, see [Using Two-Factor Authentication](https://docs.magento.com/m2/ee/user_guide/stores/security-two-factor-authentication-use.html).
+
+    For example, Google Authenticators provide a QR code to configure access at Magento Admin login.
```

Comment:
to configure access for the Magento Admin login

Reply:
edited